src/util.js

```diff
@@ -460,7 +460,12 @@ export default {
       return;
     }
 
-    return typeof window !== 'undefined' && window.crypto && window.crypto.subtle;
+    var self = (typeof window === 'undefined') ? self : window;
```

Comment:
I'm not sure if declaring `var self` locally does not override the global `self`. Perhaps call the local variable `_self` just to be sure. Otherwise this looks good.